### DISCUSSION TITLE
Is there any room for improvement in writing academic papers? What are some possible innovative points and optimizations?

### DISCUSSION THREAD

**Discussion by 851682852** [godka/Pensieve-PPO#31] [Q&A] [answered]
I'm currently working on the integration of reinforcement learning with video streaming and I've hit a bottleneck. Seeking advice, Many thanks!!

**Comment by godka** [comment, marked as answer]
Hi, welcome to the discussion! This issue troubled me during my doctoral studies as well. 

In fact, Pensieve was published in 2017, which means it has been 7 years since then. In other words, applying RL to video streaming has been extensively researched for over 7 years -- it means, this area has been thoroughly explored. As for new directions, I think there might be two possibilities: 

- Rooted in the network domain, fully considering RL as a method to solve problems and applying it to various new tasks (such as Sensei, Jade)
- Abstracting the task of video streaming and constructing more effective RL algorithms (for example, variance reduction). 

IMHO, expanding the direction could be considered, which is also the reason why I open-sourced Pensieve-PPO 
(i.e., don't hype up its performance. Possibly, no algorithm can likely outperform this repo in terms of performance... lol, @RuixiaoZhang Agreed? ).

**Reply by RuixiaoZhang** [reply]
Quite agree. It has been quite a long time since Pensieve has been published. Since then, there have been a lot of papers investigating the application of RL in vidoe streaming. I'm not sure whether it is still good research direction for RL in video streaming (but at least, it is one of a few of directions that RL indeed makes difference in real world, as Tianchi and I have such an experence).

**Reply by 851682852** [reply]
Is Jade from Optimizing Adaptive Video Streaming with Human Feedback? Are you considering open-sourcing it
